Saber/DepthBuffer.cpp

```diff
@@ -117,6 +140,10 @@ D3D12_GPU_DESCRIPTOR_HANDLE DepthBuffer::GetSrvGpuDescHandle() const {
 	return m_pSrvsRange->GetGpuHandle();
 }
 
+D3D12_GPU_DESCRIPTOR_HANDLE DepthBuffer::GetSrvGpuDescHandleWithMips() const {
+	return m_pSrvsRange->GetGpuHandle(1);
```

Comment:
better constexpr variables with understandable names. or enum, but nod magic values

Reply:
added indices for srvs